features/support/journey_helpers.rb

```diff
@@ -61,12 +74,27 @@ def select_random_upper_tier_options(carrier)
 end
 
 def select_random_lower_tier_options
+  return answer_random_lower_tier_questions unless @resource_object == :new_registration
+
+  i = rand(2)
+  if i == 0
```

Comment:
I would rather "Help me decide" was the more common option here so that we can test the flows more frequently.

Consider `if i != 0`?

Reply:
Same as https://github.com/DEFRA/waste-carriers-acceptance-tests/pull/266/files#r405629356

Reply:
See response to other comment, I'm happy with this now :)